web/src/lib/utils/analytics.ts

```diff
@@ -2594,6 +2620,150 @@ export default {
         },
       },
     },
+    gen_guidance_stepper_initiated: {
+      arrive: {
+        general_guidance_initiated: () => {
+          eventRunner.track({
+            event: "env_requirements_assessment_started",
+            legacy_event_action: "arrive",
+            legacy_event_category: "environmental_requirements_assessment",
+            legacy_event_label: "general_guidance_initiated",
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_step_category: {
+      click: {
+        general_guidance_step: (step: number, category: string) => {
+          eventRunner.track({
+            event: "env_stepper_step_viewed",
+            legacy_event_action: "click",
+            legacy_event_category: "environmental_requirements_stepper",
+            legacy_event_label: `step_${step}_${category}` as LegacyEventLabel,
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_save_see_results: {
+      click: {
+        general_guidance_save_see_results: () => {
+          eventRunner.track({
+            event: "env_final_cta_clicked",
+            legacy_event_action: "click",
+            legacy_event_category: "environmental_requirements_final_cta",
+            legacy_event_label: "save_see_results_clicked",
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_permits_required_displayed: {
+      appears: {
+        general_guidance_permits_req_displayed: () => {
+          eventRunner.track({
+            event: "env_permits_required_displayed",
+            legacy_event_action: "appears",
+            legacy_event_category: "environmental_requirements_results",
+            legacy_event_label: "permits_required_displayed",
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_no_permits_displayed: {
+      appears: {
+        general_guidance_no_permits_displayed: () => {
+          eventRunner.track({
+            event: "env_no_permits_displayed",
+            legacy_event_action: "appears",
+            legacy_event_category: "environmental_requirements_results",
+            legacy_event_label: "no_permits_needed_displayed",
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_env_req_edits: {
+      click: {
+        general_guidance_env_req_edits: () => {
+          eventRunner.track({
+            event: "env_responses_edit_started",
+            legacy_event_action: "click",
+            legacy_event_category: "environmental_requirements_edit",
+            legacy_event_label: "edit_responses_clicked",
+          });
+        },
+      },
+    },
+    gen_guidance_stepper_contact_info_engagement: {
+      mouseover: {
+        general_guidance_contact_info_engage: () => {
+          eventRunner.track({
+            event: "env_contact_info_engaged",
+            legacy_event_action: "mouseover",
```

Comment:
@williamavasquez @somabadri This is more of a question for product. Is mouseover the best choice here?

The only other place in the application we're firing events on mouseover is for the tool-tip. This seems like a new pattern for us, and I'm curious what we're trying to gather by using this as opposed to tracking when a user actually engages with a field via focus.

Can y'all confirm with this is really what we want to do?